phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -3206,6 +3219,9 @@ public void upgradeSystemTables(final String url, final Properties props) throws
             try {
                 metaConnection.createStatement().executeUpdate(getMutexDDL());
             } catch (NewerTableAlreadyExistsException e) {} catch (TableAlreadyExistsException e) {}
+            try {
```

Comment:
nit: Can we refactor and extract out these dupe pieces of code into a method?

Reply:
No sure about any value of extracting this simple try catch into a method.

Reply:
Sorry for not being clear, we can refactor the method `upgradeSystemTables` and `createOtherSystemTables` in this one.

Reply:
@karanmehta93, there is really no duplicate pieces of code here. Note that these methods have different try-catch logic